README.md

```diff
@@ -130,9 +130,9 @@ npm run dev
 - Update `ably` section under `config/broadcasting.php` with `'token_expiry' => env('ABLY_TOKEN_EXPIRY', 3600)`
 
 <a name="migrate-pusher-to-ably"></a>
-## Migrating from old AblyBroadcaster
-- The current Ably broadcaster is fully compatible with the old Pusher based AblyBroadcaster.
--  The only difference is for **Leaving the channel**, you should use [Ably Channel Namespaces](https://ably.com/docs/general/channel-rules-namespaces) conventions
+## Migrating from pusher/pusher-compatible broadcasters
+- The current Ably broadcaster is fully compatible with the [pusher](https://laravel.com/docs/9.x/broadcasting#pusher-channels), [old AblyBroadcaster](https://laravel.com/docs/9.x/broadcasting#ably) and [pusher compatible open source broadcasters](https://laravel.com/docs/9.x/broadcasting#open-source-alternatives).
```

Comment:
Is the official name of the broadcaster `AblyBroadcaster` or Ably broadcaster? I think we should stick to a single naming convention.

Reply:
Fixed